modules/ROOT/pages/install-gke.adoc

```diff
@@ -0,0 +1,64 @@
+= Luna Streaming Installation on Google Kubernetes Environment
+
+This document will cover installation of Luna Streaming on the Google Kubernetes Environment, or GKE. 
+
+== Prerequisites 
+
+* Helm version 3
+* A Kubernetes cluster 
+* No less than 16 gigs of RAM and 8 virtual processor cores (4 physical cores)
+
+For more on Helm, see https://helm.sh/docs/[Helm documentation].
+For more on Minikube, see https://minikube.sigs.k8s.io/docs/start/[Minikube documentation]. 
+
+== Provision Your GKE Cluster
+
+Before you can install the Helm chart, you need to configure the storage class settings for your cloud provider. The handling of storage varies from cloud provider to cloud provider.
+
+Create a new file called `storage_values.yaml` for the storage class settings. This allows you to match the optimum storage type to the volume.
+If you have specific storage class requirement, for example fixed IOPS disks in AWS, you can have the chart configure the storage classes for you.
+Here is an example `storage-values.yaml` for GKE.
+
+----
+# For GKE
+# default_storage:
+#   provisioner: kubernetes.io/gce-pd
+#   type: pd-ssd
+#   fsType: ext4
+#   extraParams:
+#      replication-type: none
+----
+
+See this https://github.com/datastax/pulsar-helm-chart/blob/master/helm-chart-sources/pulsar/values.yaml[values file] for more details on the settings in `values.yaml`.
+
+Additional provisioning set-up options are available https://docs.k8ssandra.io/install/gke/[here].
+
+== Install Luna Streaming to GKE
+
+Once you have added your storage settings to the `storage-values.yaml` file, install the Helm chart. 
+
+. Create the namespace; in this example, we use `pulsar`.
++
+----
+kubectl create namespace pulsar
+----
+
+. Run this Helm command using the `storage-values.yaml` provisioning:
++
+----
+helm install pulsar datastax-pulsar/pulsar --namespace pulsar --values storage_values.yaml --create-namespace
+----
+
+. To avoid having to specify the `pulsar` namespace on each subsequent command, set the namespace context. Example:
++
+----
+kubectl config set-context $(kubectl config current-context) --namespace=pulsar
+----
+
+. Execute `kubectl get pods -A` to view your running pods. 
+. Once your pods are up and running, continue to xref:quickstart-helm-installs.adoc#manage-pulsar-cluster[Accessing Pulsar Cluster in Cloud] to access and manage your cluster. 
+
+== Next
```

Comment:
As a general comment, especially after I also took a look at the other 2 cloud provider specific documents, I don' think these documents are really necessary beyond the existing Helm Chart doc: https://docs.datastax.com/en/luna/streaming/2.8/quickstart-helm-installs.html

The only difference in these documents is about the storage class difference which is already covered by the Helm Chart doc. Other than that, there is nothing cloud provider specific stuff. 

So unless we want to describe the procedure differences about how to launch the K8s clusters in these different cloud providers, I don't think there are much value added by having these 3 documents.

Reply:
@mendonk Please make these changes or resolve if you already have. Thanks!